packages/map-template/src/hooks/useMapBoundsDeterminer.js

```diff
@@ -201,7 +201,7 @@ function mapboxGotoBBox(bbox, mapsIndoorsInstance, paddingBottom, paddingLeft, z
     // need to be able to use pitch and bearing in one go,
     // and we want to turn of panning animation.
     mapboxMap.fitBounds(bbox, {
-        pitch: pitch || 0,
+        pitch: pitch || mapboxMap.getPitch() || 0,
```

Comment:
To make it work 😂 

On a more serious note, it's because of the View mode switch. When starting in initial 3D mode, the MIMap itself tilts the map to a value of 45. If we don't apply this check here, the map is tilted back to 0 instantly.